docs/rules/no-duplicate-attr-inheritance.md

```diff
@@ -12,10 +12,10 @@ since: v7.0.0
 
 ## :book: Rule Details
 
-This rule aims to prevent duplicate attribute inheritance.  
+This rule aims to prevent duplicate attribute inheritance.
 This rule to warn to apply `inheritAttrs: false` when it detects `v-bind="$attrs"` being used.
```

Comment:
```suggestion
This rule aims to prevent duplicate attribute inheritance.  
This rule suggests applying `inheritAttrs: false` when it detects `v-bind="$attrs"` being used.
```